pkg/libvmi/vm.go

```diff
@@ -59,6 +59,12 @@ func NewVirtualMachine(vmi *v1.VirtualMachineInstance, opts ...VMOption) *v1.Vir
 	return vm
 }
 
+func WithRunStrategy(runStrategy v1.VirtualMachineRunStrategy) VMOption {
+	return func(vm *v1.VirtualMachine) {
+		vm.Spec.RunStrategy = &runStrategy
+	}
+}
```

Comment:
WDYT about also assigning `vm.Spec.Running = nil` here?
I think it makes sense as it would help get rid of "both .spec.running and .spec.runStrategy are set" error.

Reply:
I can try but what's happening in the tests is that many of them are setting running=true pre-creation https://github.com/kubevirt/kubevirt/blob/bee5cc119bfb95e6b41860da56c2ba2bbcbe28f3/tests/storage/snapshot.go#L336-L339